### PR TITLE
Move to jinja2  

### LINKED ISSUE #1: Move to aiohttp_jinja2 templates
For now mako templates used with aiohttp_mako, may be this should be changed to aiohttp_jinja2? as more mature and maintained library?

### PR DESCRIPTION
**Pull request by iho**
Closes #1

### CODE REVIEW

File: aiohttp_debugtoolbar/templates/toolbar.dbtmako [hunk 1]
```diff
@@ -66,10 +65,10 @@
     </div>
 
     ## scripts that can be included after pageload
```
Comment (jettify) [inline aiohttp_debugtoolbar/templates/toolbar.dbtmako]: This line is visible on rendered page, should be enclosed in ``{# #}``.

**Comment by jettify** [comment]
From first glance looks pretty good! But let me play PR bit more... And one more thing, could you rename templates from ``*.dbtmako`` to ``*.jinja2`` ?

File: demo/demo.py [hunk 2]
```diff
@@ -105,8 +104,6 @@ def init(loop):
     app.router.add_route('GET', '/call_ajax', call_ajax, name='call_ajax')
 
     # templates error handlers
-    app.router.add_route('GET', '/mako_exc', test_mako_exc,
```
Comment (jettify) [inline demo/demo.py]: Could you revert ``app.router.add_route('GET', '/mako_exc', test_mako_exc`` changes in demo app? I want to have both: ``mako`` and ``jinja2`` templates there to make sure that both working and exceptions generated inside template engine have meaningful representation.

File: tests/base.py [hunk 3]
```diff
@@ -27,8 +27,10 @@ def find_unused_port():
 
 
 class BaseTest(unittest.TestCase):
+
```
Comment (jettify) [inline tests/base.py]: Why did you add empty line before doc string every class? This looks unusual and bit ugly.

**Comment by jettify** [comment]
Why did you introduce so many style changes and import reorganizations not related to this PR? It is not necessary bad thing, but now it is pain to review all modifications... You could have done that in separate PR.

File: tests/base.py (hunk 3 above)
Comment (iho) [inline tests/base.py]: I got error on make test and make cov so I run https://pypi.python.org/pypi/isort and https://pypi.python.org/pypi/autopep8/

File: demo/demo.py (hunk 2 above)
Comment (iho) [inline demo/demo.py]: Can I create two version with for people who do not want istall aiohtpp_mako?

File: tests/base.py (hunk 3 above)
Comment (asvetlov) [inline tests/base.py]: We use no blank line after `class` and `def` for now, following `asyncio` code style.
If it's ok to Guido then it's perfect for us also.

**Comment by iho** [comment]
@jettify all this stuff from commit "pep8 fixes" so just do not look them.

File: demo/demo.py (hunk 2 above)
Comment (asvetlov) [inline demo/demo.py]: If you really want to allow demo run without `aiohttp_mako` installed please do conditional import.
I mean 
```
try:
    import aiohttp_mako
except ImportError:
    aiohttp_mako = None

if aiohttp_mako:
   app.router.add_route('GET', '/mako_exc', test_mako_exc, name='test_mako_exc')
```

index page for demo app should use pattern like: 
```
{% if aiohttp_mako %}
        <p>
          <a href="${app.router['test_mako_exc'].url()}">Mako Exception example</a>
        </p>
{% else %}
        <p>
          aiohttp_mako is not available, please install it by <code><pre>$ pip install aiohttp_mako</pre></code> to see how mako excetion handling works.
        </p>
{% endif %}
```

**Comment by jettify** [comment]
@iho  I've played with your PR and demo app, looks like everything working. Revert please *autopep* fixes and add ``aiohttp_mako`` in a way @asvetlov suggested.

**Comment by iho** [comment]
@jettify sommething not ok? CI cannot build because it have pep8 errors.

**Comment by iho** [comment]
Why you do not want merge this PR?

**Comment by jettify** [comment]
wait a bit, I am working on this right now :)

**Comment by jettify** [comment]
Thank you for your effort!

**Comment by iho** [comment]
Cool! :+1:
